[Iceberg] Support procedure `remove_orphan_files` (#23267)
prestodb/presto: presto-docs/src/main/sphinx/connector/iceberg.rst

    CALL iceberg.system.expire_snapshots(schema => 'schema_name', table_name => 'table_name', snapshot_ids => ARRAY[10001, 10002]);

Remove Orphan Files
^^^^^^^^^^^^^^^^^^^

> Used to remove files which are not referenced in any metadata files of an Iceberg table and can thus be considered "orphaned".

Reviewer: ```suggestion
Use to remove files which are not referenced in any metadata files of an Iceberg table.
```

Author: Thanks for the suggestion, fixed! Please take a look when available.
